[libvorbis] Build error

**Host Environment**
- Windows 11
- GCC 11.2.0
- MinGW-w64 9.0.0 (linked with ucrt) (from winlibs.com)


**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg --triplet=x64-mingw-dynamic --host-triplet=x64-mingw-dynamic install libvorbis`

**Failure logs** 
vcpkg console:
```
Computing installation plan...
The following packages will be built and installed:
    libvorbis[core]:x64-mingw-dynamic -> 1.3.7#1
Detecting compiler hash for triplet x64-mingw-dynamic...
Restored 0 packages from C:\Users\USERNAME\AppData\Local\vcpkg\archives in 172.3 us. Use --debug to see more details.
Starting package 1/1: libvorbis:x64-mingw-dynamic
Building package libvorbis[core]:x64-mingw-dynamic...
-- Using community triplet x64-mingw-dynamic. This triplet configuration is not guaranteed to succeed.
-- [COMMUNITY] Loading triplet configuration from: C:\mingw64-new\vcpkg\triplets\community\x64-mingw-dynamic.cmake
-- Using cached xiph-vorbis-v1.3.7.tar.gz.
-- Cleaning sources at C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/mingw64-new/vcpkg/downloads/xiph-vorbis-v1.3.7.tar.gz
-- Applying patch 0001-Dont-export-vorbisenc-functions.patch
-- Applying patch 0002-Fixup-pkgconfig-libs.patch
-- Using source at C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean
-- Found external ninja('1.10.2').
-- Configuring x64-mingw-dynamic
-- Building x64-mingw-dynamic-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:158 (message):
    Command failed: "C:/Program Files/CMake/bin/cmake.exe" --build . --config Debug --target install -- -v -j17
    Working Directory: C:/mingw64-new/vcpkg/buildtrees/libvorbis/x64-mingw-dynamic-dbg
    See logs for more information:
      C:\mingw64-new\vcpkg\buildtrees\libvorbis\install-x64-mingw-dynamic-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:106 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:44 (vcpkg_build_cmake)
  ports/libvorbis/portfile.cmake:17 (vcpkg_install_cmake)
  scripts/ports.cmake:145 (include)


Error: Building package libvorbis:x64-mingw-dynamic failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `git pull` and `.\vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+libvorbis
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[libvorbis]+Build+error
including:
  package: libvorbis[core]:x64-mingw-dynamic -> 1.3.7#1
    vcpkg-tool version: 2022-02-03-457b0309cefb58fae0dcb2e4c06de9bd7582967f
    vcpkg-scripts version: d1a87fc3e 2022-02-07 (7 minutes ago)
```

install-x64-mingw-dynamic-dbg-out.log
```
[1/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/lpc.c.obj -MF lib\CMakeFiles\vorbis.dir\lpc.c.obj.d -o lib/CMakeFiles/vorbis.dir/lpc.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/lpc.c
[2/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/registry.c.obj -MF lib\CMakeFiles\vorbis.dir\registry.c.obj.d -o lib/CMakeFiles/vorbis.dir/registry.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/registry.c
[3/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/window.c.obj -MF lib\CMakeFiles\vorbis.dir\window.c.obj.d -o lib/CMakeFiles/vorbis.dir/window.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/window.c
[4/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/analysis.c.obj -MF lib\CMakeFiles\vorbis.dir\analysis.c.obj.d -o lib/CMakeFiles/vorbis.dir/analysis.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/analysis.c
[5/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/synthesis.c.obj -MF lib\CMakeFiles\vorbis.dir\synthesis.c.obj.d -o lib/CMakeFiles/vorbis.dir/synthesis.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/synthesis.c
[6/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/lsp.c.obj -MF lib\CMakeFiles\vorbis.dir\lsp.c.obj.d -o lib/CMakeFiles/vorbis.dir/lsp.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/lsp.c
[7/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/envelope.c.obj -MF lib\CMakeFiles\vorbis.dir\envelope.c.obj.d -o lib/CMakeFiles/vorbis.dir/envelope.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/envelope.c
[8/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/floor0.c.obj -MF lib\CMakeFiles\vorbis.dir\floor0.c.obj.d -o lib/CMakeFiles/vorbis.dir/floor0.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/floor0.c
[9/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/codebook.c.obj -MF lib\CMakeFiles\vorbis.dir\codebook.c.obj.d -o lib/CMakeFiles/vorbis.dir/codebook.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/codebook.c
[10/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/res0.c.obj -MF lib\CMakeFiles\vorbis.dir\res0.c.obj.d -o lib/CMakeFiles/vorbis.dir/res0.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/res0.c
[11/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/mdct.c.obj -MF lib\CMakeFiles\vorbis.dir\mdct.c.obj.d -o lib/CMakeFiles/vorbis.dir/mdct.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/mdct.c
[12/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/info.c.obj -MF lib\CMakeFiles\vorbis.dir\info.c.obj.d -o lib/CMakeFiles/vorbis.dir/info.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/info.c
[13/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/mapping0.c.obj -MF lib\CMakeFiles\vorbis.dir\mapping0.c.obj.d -o lib/CMakeFiles/vorbis.dir/mapping0.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/mapping0.c
[14/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/block.c.obj -MF lib\CMakeFiles\vorbis.dir\block.c.obj.d -o lib/CMakeFiles/vorbis.dir/block.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/block.c
[15/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/floor1.c.obj -MF lib\CMakeFiles\vorbis.dir\floor1.c.obj.d -o lib/CMakeFiles/vorbis.dir/floor1.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/floor1.c
[16/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/smallft.c.obj -MF lib\CMakeFiles\vorbis.dir\smallft.c.obj.d -o lib/CMakeFiles/vorbis.dir/smallft.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/smallft.c
[17/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/lookup.c.obj -MF lib\CMakeFiles\vorbis.dir\lookup.c.obj.d -o lib/CMakeFiles/vorbis.dir/lookup.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/lookup.c
[18/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/bitrate.c.obj -MF lib\CMakeFiles\vorbis.dir\bitrate.c.obj.d -o lib/CMakeFiles/vorbis.dir/bitrate.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/bitrate.c
[19/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/sharedbook.c.obj -MF lib\CMakeFiles\vorbis.dir\sharedbook.c.obj.d -o lib/CMakeFiles/vorbis.dir/sharedbook.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/sharedbook.c
[20/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/psy.c.obj -MF lib\CMakeFiles\vorbis.dir\psy.c.obj.d -o lib/CMakeFiles/vorbis.dir/psy.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/psy.c
[21/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbisfile_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbisfile.dir/vorbisfile.c.obj -MF lib\CMakeFiles\vorbisfile.dir\vorbisfile.c.obj.d -o lib/CMakeFiles/vorbisfile.dir/vorbisfile.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/vorbisfile.c
[22/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbisenc_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbisenc.dir/vorbisenc.c.obj -MF lib\CMakeFiles\vorbisenc.dir\vorbisenc.c.obj.d -o lib/CMakeFiles/vorbisenc.dir/vorbisenc.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/vorbisenc.c
[23/27] C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -Dvorbis_EXPORTS -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/include -IC:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib -isystem C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/include -g -MD -MT lib/CMakeFiles/vorbis.dir/vorbisenc.c.obj -MF lib\CMakeFiles\vorbis.dir\vorbisenc.c.obj.d -o lib/CMakeFiles/vorbis.dir/vorbisenc.c.obj -c C:/mingw64-new/vcpkg/buildtrees/libvorbis/src/v1.3.7-6c4ca9c573.clean/lib/vorbisenc.c
[24/27] cmd.exe /C "cd . && C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -g  C:\mingw64-new\vcpkg\buildtrees\libvorbis\src\v1.3.7-6c4ca9c573.clean\win32\vorbis.def -shared -o lib\libvorbis.dll -Wl,--out-implib,lib\libvorbis.dll.a -Wl,--major-image-version,0,--minor-image-version,0 lib/CMakeFiles/vorbis.dir/mdct.c.obj lib/CMakeFiles/vorbis.dir/smallft.c.obj lib/CMakeFiles/vorbis.dir/block.c.obj lib/CMakeFiles/vorbis.dir/envelope.c.obj lib/CMakeFiles/vorbis.dir/window.c.obj lib/CMakeFiles/vorbis.dir/lsp.c.obj lib/CMakeFiles/vorbis.dir/lpc.c.obj lib/CMakeFiles/vorbis.dir/analysis.c.obj lib/CMakeFiles/vorbis.dir/synthesis.c.obj lib/CMakeFiles/vorbis.dir/psy.c.obj lib/CMakeFiles/vorbis.dir/info.c.obj lib/CMakeFiles/vorbis.dir/floor1.c.obj lib/CMakeFiles/vorbis.dir/floor0.c.obj lib/CMakeFiles/vorbis.dir/res0.c.obj lib/CMakeFiles/vorbis.dir/mapping0.c.obj lib/CMakeFiles/vorbis.dir/registry.c.obj lib/CMakeFiles/vorbis.dir/codebook.c.obj lib/CMakeFiles/vorbis.dir/sharedbook.c.obj lib/CMakeFiles/vorbis.dir/lookup.c.obj lib/CMakeFiles/vorbis.dir/bitrate.c.obj lib/CMakeFiles/vorbis.dir/vorbisenc.c.obj  C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/debug/lib/libogg.dll.a  -lm  -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 && cd ."
FAILED: lib/libvorbis.dll lib/libvorbis.dll.a 
cmd.exe /C "cd . && C:\mingw64-new\bin\x86_64-w64-mingw32-gcc.exe -g  C:\mingw64-new\vcpkg\buildtrees\libvorbis\src\v1.3.7-6c4ca9c573.clean\win32\vorbis.def -shared -o lib\libvorbis.dll -Wl,--out-implib,lib\libvorbis.dll.a -Wl,--major-image-version,0,--minor-image-version,0 lib/CMakeFiles/vorbis.dir/mdct.c.obj lib/CMakeFiles/vorbis.dir/smallft.c.obj lib/CMakeFiles/vorbis.dir/block.c.obj lib/CMakeFiles/vorbis.dir/envelope.c.obj lib/CMakeFiles/vorbis.dir/window.c.obj lib/CMakeFiles/vorbis.dir/lsp.c.obj lib/CMakeFiles/vorbis.dir/lpc.c.obj lib/CMakeFiles/vorbis.dir/analysis.c.obj lib/CMakeFiles/vorbis.dir/synthesis.c.obj lib/CMakeFiles/vorbis.dir/psy.c.obj lib/CMakeFiles/vorbis.dir/info.c.obj lib/CMakeFiles/vorbis.dir/floor1.c.obj lib/CMakeFiles/vorbis.dir/floor0.c.obj lib/CMakeFiles/vorbis.dir/res0.c.obj lib/CMakeFiles/vorbis.dir/mapping0.c.obj lib/CMakeFiles/vorbis.dir/registry.c.obj lib/CMakeFiles/vorbis.dir/codebook.c.obj lib/CMakeFiles/vorbis.dir/sharedbook.c.obj lib/CMakeFiles/vorbis.dir/lookup.c.obj lib/CMakeFiles/vorbis.dir/bitrate.c.obj lib/CMakeFiles/vorbis.dir/vorbisenc.c.obj  C:/mingw64-new/vcpkg/installed/x64-mingw-dynamic/debug/lib/libogg.dll.a  -lm  -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 && cd ."
c:/mingw64-new/bin/../lib/gcc/x86_64-w64-mingw32/11.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\mingw64-new\vcpkg\buildtrees\libvorbis\src\v1.3.7-6c4ca9c573.clean\win32\vorbis.def:3: syntax error
c:/mingw64-new/bin/../lib/gcc/x86_64-w64-mingw32/11.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe:C:\mingw64-new\vcpkg\buildtrees\libvorbis\src\v1.3.7-6c4ca9c573.clean\win32\vorbis.def: file format not recognized; treating as linker script
c:/mingw64-new/bin/../lib/gcc/x86_64-w64-mingw32/11.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe:C:\mingw64-new\vcpkg\buildtrees\libvorbis\src\v1.3.7-6c4ca9c573.clean\win32\vorbis.def:2: syntax error
collect2.exe: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
```

**Additional context**
The first ten lines of the apparently misbehaving vorbis.def
```
; vorbis.def
; 
LIBRARY
EXPORTS
_floor_P
_mapping_P
_residue_P
;
vorbis_info_init
vorbis_info_clear
```

The static version of the libraries (with --triplet=x64-mingw-static) build successfully.

--- a/ports/libvorbis/0003-def-mingw-compat.patch
+++ b/ports/libvorbis/0003-def-mingw-compat.patch
@@ -1,0 +1,33 @@
+diff --git a/win32/vorbis.def b/win32/vorbis.def
+index 1310b6c..de14385 100644
+--- a/win32/vorbis.def
++++ b/win32/vorbis.def
+@@ -1,6 +1,5 @@
+ ; vorbis.def
+ ; 
+-LIBRARY
+ EXPORTS
+ _floor_P
+ _mapping_P
+diff --git a/win32/vorbisenc.def b/win32/vorbisenc.def
+index 79af064..40a3e39 100644
+--- a/win32/vorbisenc.def
++++ b/win32/vorbisenc.def
+@@ -1,6 +1,5 @@
+ ; vorbisenc.def
+ ;
+-LIBRARY
+ 
+ EXPORTS
+ vorbis_encode_init
+diff --git a/win32/vorbisfile.def b/win32/vorbisfile.def
+index 4dc5549..243795d 100644
+--- a/win32/vorbisfile.def
++++ b/win32/vorbisfile.def
+@@ -1,6 +1,5 @@
+ ; vorbisfile.def
+ ;
+-LIBRARY
+ EXPORTS
+ ov_clear
+ ov_open

--- a/ports/libvorbis/portfile.cmake
+++ b/ports/libvorbis/portfile.cmake
@@ -7,6 +7,7 @@ vcpkg_from_github(
     PATCHES
         0001-Dont-export-vorbisenc-functions.patch
         0002-Fixup-pkgconfig-libs.patch
+        0003-def-mingw-compat.patch
 )
 
 vcpkg_configure_cmake(

--- a/ports/libvorbis/portfile.cmake
+++ b/ports/libvorbis/portfile.cmake
@@ -12,7 +12,6 @@ vcpkg_from_github(
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
-    PREFER_NINJA
 )
 
 vcpkg_cmake_install()

--- a/ports/libvorbis/portfile.cmake
+++ b/ports/libvorbis/portfile.cmake
@@ -10,22 +10,17 @@ vcpkg_from_github(
         0003-def-mingw-compat.patch
 )
 
-vcpkg_configure_cmake(
-    SOURCE_PATH ${SOURCE_PATH}
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
     PREFER_NINJA
 )
 
-vcpkg_install_cmake()
-vcpkg_fixup_cmake_targets(
-    CONFIG_PATH lib/cmake/Vorbis
-    TARGET_PATH share/Vorbis
-)
-
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
-
-# Handle copyright
-configure_file(${SOURCE_PATH}/COPYING ${CURRENT_PACKAGES_DIR}/share/${PORT}/copyright COPYONLY)
-
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(PACKAGE_NAME Vorbis CONFIG_PATH "lib/cmake/Vorbis")
+vcpkg_fixup_pkgconfig()
 vcpkg_copy_pdbs()
 
-vcpkg_fixup_pkgconfig()
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+
+# Handle copyright
+file(INSTALL "${SOURCE_PATH}/COPYING" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

--- a/ports/libvorbis/portfile.cmake
+++ b/ports/libvorbis/portfile.cmake
@@ -22,5 +22,5 @@ vcpkg_copy_pdbs()
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
-# Handle copyright
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
 file(INSTALL "${SOURCE_PATH}/COPYING" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

--- a/ports/libvorbis/usage
+++ b/ports/libvorbis/usage
@@ -1,0 +1,13 @@
+The package libvorbis provides CMake targets:
+
+    # Vorbis reference encoder and decoder, low-level API
+    find_package(Vorbis CONFIG REQUIRED)
+    target_link_libraries(main PRIVATE Vorbis::vorbis)
+
+    # Audio stream decoding and basic manipulation, high-level API
+    find_package(Vorbis CONFIG REQUIRED)
+    target_link_libraries(main PRIVATE Vorbis::vorbisfile)
+
+    # Convenience API for setting up an encoding environment
+    find_package(Vorbis CONFIG REQUIRED)
+    target_link_libraries(main PRIVATE Vorbis::vorbisenc)

--- a/ports/libvorbis/vcpkg.json
+++ b/ports/libvorbis/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "libvorbis",
   "version-string": "1.3.7",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Ogg Vorbis is a fully open, non-proprietary, patent-and-royalty-free, general-purpose compressed audio format",
   "homepage": "https://github.com/xiph/vorbis",
   "license": "BSD-3-Clause",

--- a/ports/libvorbis/vcpkg.json
+++ b/ports/libvorbis/vcpkg.json
@@ -1,11 +1,19 @@
 {
   "name": "libvorbis",
-  "version-string": "1.3.7",
+  "version": "1.3.7",
   "port-version": 2,
   "description": "Ogg Vorbis is a fully open, non-proprietary, patent-and-royalty-free, general-purpose compressed audio format",
   "homepage": "https://github.com/xiph/vorbis",
   "license": "BSD-3-Clause",
   "dependencies": [
-    "libogg"
+    "libogg",
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
   ]
 }

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4126,7 +4126,7 @@
     },
     "libvorbis": {
       "baseline": "1.3.7",
-      "port-version": 1
+      "port-version": 2
     },
     "libvpx": {
       "baseline": "1.10.0",

--- a/versions/l-/libvorbis.json
+++ b/versions/l-/libvorbis.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "75830541f31d64e059859f1348337eeae5571ae7",
+      "git-tree": "c4c646a26ce3e6ad63992d3e4a55cb185159b23c",
       "version": "1.3.7",
       "port-version": 2
     },

--- a/versions/l-/libvorbis.json
+++ b/versions/l-/libvorbis.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "75830541f31d64e059859f1348337eeae5571ae7",
+      "version": "1.3.7",
+      "port-version": 2
+    },
+    {
       "git-tree": "54dcb5ab354422e9518a905d501ccc22dcb69098",
       "version-string": "1.3.7",
       "port-version": 1

